LLama/Native/LLavaImageEmbed.cs

```diff
@@ -7,6 +7,13 @@ namespace LLama.Native;
 [StructLayout(LayoutKind.Sequential)]
 public unsafe struct LLavaImageEmbed
 {
+    /// <summary>
+    /// The embeddings of the embedded image. Should be a float[]
```

Comment:
```suggestion
    /// The embeddings of the embedded image.
```
Span or Memory or any other buffer type is acceptable too!